code/modules/reagents/reagent_containers/blood_pack.dm

```diff
@@ -101,21 +82,25 @@
 	blood_type = BLOOD_TYPE_UNIVERSAL
 
 /obj/item/reagent_containers/blood/attackby(obj/item/tool, mob/user, list/modifiers, list/attack_modifiers)
```

Comment:
Can't believe ya took the opportunity to clean this up and didn't make it item interaction, smh